### PR TITLE
#156839621 Update README and requirements.txt

### PR DESCRIPTION
**Pull request by SoniaRMK**
#### What does this PR do?
Have the README updated with correct instructions

#### Description of Task to be completed?
Help a new user to quickly set up the project on their localhost

#### How should this be manually tested?
Follow the instructions step by step

#### Any background context you want to provide?
The original README had wrong instructions

#### What are the relevant pivotal tracker stories?
#156839621

@rogerokello 
@Rwothoromo 
@emugaya 
@kasigazibrian 
@louiCoder

### CODE REVIEW

**Comment by louiCoder** [comment]
@SoniaRMK i have looked through the changes and everything looks fine. lets wait and see what others think and later on @emugaya will confirm.

**Comment by Rwothoromo** [comment]
@SoniaRMK , the readme LGTM.

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,9 @@ hc/local_settings.py
...
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+
+hc-venv/
+hc/.local_settings
+
+.vscode
```
Comment (emugaya) [inline .gitignore]: Add new line to end of file

File: README.md [hunk 2]
```diff
@@ -111,21 +120,19 @@ configuration from environment variables like so:
...
-    AWS_SES_ACCESS_KEY_ID = "put-access-key-here"
-    AWS_SES_SECRET_ACCESS_KEY = "put-secret-access-key-here"
-    AWS_SES_REGION_NAME = 'us-east-1'
-    AWS_SES_REGION_ENDPOINT = 'email.us-east-1.amazonaws.com'
+links and alerts to users. You will likely need to tweak email configuration 
+in `hc/local_settings.py` before emails work:
+
+    EMAIL_HOST = 'smtp.sendgrid.net'
+    EMAIL_HOST_USER = '<sendgrid_username>'
+    EMAIL_HOST_PASSWORD = '<sendgrid_api_key>'
```
Comment (emugaya) [inline README.md]: Are you saving the key in and environment variable?

**Comment by emugaya** [comment]
Name the PR in line with the naming convention in the guide I shared.

File: README.md (hunk 2 above)
Comment (SoniaRMK) [reply README.md]: Basically with send grid, after setting up via their website, I use the password generated as EMAIL_HOST_PASSWORD, the key is what is used to generate the password. No key is saved anywhere in the HC Project.

Comment (emugaya) [reply README.md]: Cool. Has slack been integrated?

Comment (SoniaRMK) [reply README.md]: @Slack, yes however, under the github settings via slack app, I need to add the particular repo we want to be receiving notification from and for me to add the repo, I need admin rights to the repo

**Pull request closed without merging**